src/gatsby-ssr.js

```diff
@@ -1,23 +1,61 @@
-const React = require('react')
+const React = require('react');
 
 exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
-  const domain = pluginOptions.domain || 'cdn.simpleanalytics.io'
-  const scriptName = domain === 'cdn.simpleanalytics.io' ? 'hello.js' : 'app.js'
-  const scriptUrl = `https://${domain}/${scriptName}`
+  const domain = pluginOptions.domain || 'cdn.simpleanalytics.io';
+  const scriptName =
+    domain === 'cdn.simpleanalytics.io' ? 'hello.js' : 'app.js';
+  const scriptUrl = `https://${domain}/${scriptName}`;
 
   const options = {
-    key: 'plugin-simpleanalytics',
     src: scriptUrl,
-    async: true,
-    defer: true
-  }
+    async: true
+  };
 
   if (pluginOptions.metomic) {
-    options.type = 'text/x-metomic'
-    options['data-micropolicy'] = pluginOptions.metomic
+    options.type = 'text/x-metomic';
+    options.metomic = pluginOptions.metomic;
   }
 
   setHeadComponents([
-    React.createElement('script', options)
-  ])
-}
+    React.createElement('script', {
+      key: 'plugin-simpleanalytics',
+      dangerouslySetInnerHTML: {
+        __html: loadScript(domain, options)
+      }
+    })
+  ]);
+};
+
+// this script runs when the page loads, it does the following;
+// 1. Checks the doNotTrack option, and if set then it aborts and does nothing
+// 2. Creates a script tag for the Simple Analytics script
+// 3. Sets all the options properties on the script tag
+// 4. Inserts the script tag before the first other script in the document
+const loadScript = (domain, options) => {
+  return `!(function(s, i, m, p, l, e) {
+    if (
```

Comment:
We could do something like this:

```js
var dnt = s.doNotTrack || m.doNotTrack || m.msDoNotTrack
if (['1', 'yes'].indexOf(dnt) > -1) return console.warn('Simple Analytics: Not loading script when doNotTrack is enabled');
```

Reply:
I assumed the weak equality check on the article I mentioned was intentional. Maybe in some browsers the 1 is a string and others a number? In which case this might not work... Any idea?

Reply:
From the spec it's not super clear: https://www.w3.org/TR/tracking-dnt/#expression-format but on MDN it's pretty clear: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack

There seem to be no exceptions to browsers using a string or not. They all use a string with either `yes` or `1`. https://caniuse.com/#search=dnt

I would keep the strong equality check because that's the only way browsers do it.

<img width="337" alt="CleanShot 2019-07-17 at 12 44 15@2x" src="https://user-images.githubusercontent.com/1079135/61369582-b364de80-a890-11e9-9429-e244f09e3c36.png">